### PR TITLE
fix(amf): ICS IE not required for ICS

### PR DESCRIPTION
**Pull request by Sathyaj27**
Signed-off-by: Sathyaj27 <sathya.jayadev@wavelabs.ai>

Fix For #13306 
## Summary
-Removed check for ICS IE

## Test Plan
- Tested on spirent.
ICS

## Additional Information

- [ ] This change is not backwards-breaking

### CODE REVIEW

File: lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp [hunk 1]
```diff
@@ -1030,11 +1030,7 @@ status_code_e amf_app_handle_pdu_session_accept(
                                &ue_context->amf_context._security);
   if (bytes > 0) {
     buffer->slen = bytes;
-    if (ue_context->ue_context_request) {
-      pdu_session_resource_setup_request(ue_context, ue_id, smf_ctx, buffer);
-    } else {
-      initial_context_setup_request(ue_id, &ue_context->amf_context, buffer);
-    }
+    pdu_session_resource_setup_request(ue_context, ue_id, smf_ctx, buffer);
```
Comment (pruthvihebbani) [inline lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp:1033]: Can you add comment on why the check is no longer needed?

**Review by pruthvihebbani — CHANGES_REQUESTED** [review]

File: lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp (hunk 1 above)
Comment (Sathyaj27) [reply lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp:1033]: ICS is now clubbed with session creation when UE Context Request IE not present.
In earlier behavior ICS was rejected if UE has not sent the “UE Context Request IE”.

**Review by pruthvihebbani — APPROVED** [review]
lgtm

**Review by panyogesh — APPROVED** [review]
LGTM

**Pull request merged**